Solutions/Menes.Hosting.AspNetCore/Menes/Internal/HttpRequestParameterBuilder.cs

```diff
@@ -621,8 +621,11 @@ private bool TryGetParameterFromQuery(
                         parameter.Name);
                 }
 
-                result = this.ConvertValue(parameter.Schema, value[0]);
-                return true;
+                if (value[0] is not null)
```

Comment:
If you use a more positive pattern:

```cs
value[0] is string value)
```

you can then replace the `value[0]!` with just `value`) enabling the compiler to continue to check that your nullability handling makes sense (instead of just overruling it with `!`).